contrib/arch/d1x-rebirth/d1x-rebirth.install

```diff
@@ -9,10 +9,10 @@ descent.pig
 
 You don't own the full game, yet?
 Just visit GOOD OLD GAMES via:
-https://www.gog.com/game/descent/pp/fc074d501302eb2b93e2554793fcaf50b3bf7291
+http://www.gog.com/en/gamecard/descent_1_descent_2/pp/fc074d501302eb2b93e2554793fcaf50b3bf7291
```

Comment:
Your link is out of date.  The URL in the Rebirth repository is correct.

Reply:
Oops, thanks for the heads-up.

Reply:
Well, none of the above links actually work here, this is what I have found:
https://www.gog.com/game/descent
Should we use that from now on?

Reply:
It looks like they changed the format of their partner link.  However, since the `INSTALL.markdown` instructions are not using a partner link, I think using a non-partner link here is right for consistency.  Yes, use the bare form that you proposed.